site/docs/components/icon/examples.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Icon
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="icon" exampleName="BasicIcon">
+    ### Basic Icon
+
+    Icons should be individually imported and used from the `@salt-ds/icons`
+    package
+
+  </LivePreview>
+  <LivePreview componentName="icon" exampleName="Size">
+    ### Icon Size
+
+    You can change the size of each icon using the `size` property which acts as a multiplier for the base icon’s size, as described in the [Scale Foundation](/salt/foundations/scale). The base size of Icon is based on the active density as described in the [Size Foundation](/salt/foundations/size).
+
+  </LivePreview>
+  <LivePreview componentName="icon" exampleName="SolidIcons">
```

Comment:
Examples may need a bit update to match the names. e.g. "Solid Icons" gives both solid and not solid. "Size" one may benefit from some size wording or a radio to toggle.